pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -921,6 +923,16 @@ cdef class MultiIndexHashTable(HashTable):
                     "hash collision\nlocs:\n{}\n"
                     "result:\n{}\nmi:\n{}".format(alocs, result, mi))
 
+    def _check_for_collision(self, Py_ssize_t loc, object label):
```

Comment:
make this ``cdef``, its hitting overhead because it has to go to python and back to cython